api/controllers/fda_applications.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const FdaApplication = require('../models/fda_application');
+
+function getFdaApplication(req, res) {
+  const id = req.swagger.params.id.value;
+
+  return new FdaApplication({ id: id }).fetch({
+      withRelated: ['fda_approvals', 'organisation'],
+    })
+    .catch((err) => {
+      res.finish();
+      throw err;
+    })
```

Comment:
The `catch` should be the last thing we do in the promise, as a fallback for any error that happened (e.g. if `res.json()` on line 17 throws an error, this code will hang).